services/director-v2/src/simcore_service_director_v2/cli/_core.py

```diff
@@ -87,7 +88,7 @@ async def async_project_save_state(project_id: ProjectID, save_attempts: int) ->
     async with _initialized_app() as app:
         projects_repository: ProjectsRepository = get_repository(
             app, ProjectsRepository
-        )
+        )  # type: ignore
```

Comment:
Question: what makes you choose ignore over casting?
Also we should refactor ```get_repository``` to return type T, that would fix this problem.